app/controllers/api/v1/moves_controller.rb

```diff
@@ -123,6 +127,36 @@ def move
       def updater
         @updater ||= Moves::Updater.new(move, update_move_params)
       end
+
+      def included_relationships
+        # Excludes potentially many court hearing documents to reduce the request size. This was requested specifically by the frontend team.
+        MoveSerializer::SUPPORTED_RELATIONSHIPS + include_params - %w[court_hearings]
```

Comment:
maybe there has already been discussion about this, but I think it would be better to:

* if include_params is specified in the query, then use that (filtered to all support relationships, so that you couldn't include e.g. users passwords)
* if include_params is not specified in the query then go with the current default: `MoveSerializer::SUPPORTED_RELATIONSHIPS - %w[court_hearings]`

That way we can start moving clients towards requesting what they need, rather than always give everything

Reply:
yes, I like that, I think it helps to support the include param and it's easy to explain.

Reply:
my only concern is, the FE will be forced to pass all the `include` params at once